pkg/cmd/admin/groups/new.go

```diff
@@ -49,7 +49,17 @@ func NewCmdNewGroup(name, fullName string, f *clientcmd.Factory, out io.Writer)
 				kcmdutil.CheckErr(kcmdutil.UsageError(cmd, err.Error()))
 			}
 
-			kcmdutil.CheckErr(options.AddGroup())
+			if users, err := options.AddGroup(); err != nil {
+				kcmdutil.CheckErr(err)
+			} else {
+				var msg string
+				if len(users) == 0 {
+					msg = fmt.Sprintf("Group %q created.", options.Group)
+				} else {
+					msg = fmt.Sprintf("Group %q created with users %q.", options.Group, users)
```

Comment:
Technically you know it was successful because it returned a `0` exit code.  The whole point of the change is to be verbose, and the list of user's will be de-duped as well.  That being said, I will investigate @stevekuznetsov's suggestion.